### PR TITLE
Bump CAPI version to 15.0

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## What does this change?

Removes the stories API which isn't used anywhere.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[Remove stories in frontend and MAPI](https://trello.com/c/84fvMAvW/815-remove-stories-in-frontend-and-mapi)

**Comment by gtrufitt** [comment]
Do you need to check in a new CAPI snapshot or something?

**Comment by regiskuckaertz** [comment]
Oh, I forgot to regenerate the data files. Will do that next 👍

File: data/database/b9573b35ed0df9269e70a35efff6fa898500cfba19e73d00629702972f400a4f [hunk 1]
```diff
@@ -0,0 +1 @@
+{"message":"The security token included in the request is expired"}
```
Comment (gtrufitt) [inline data/database/b9573b35ed0df9269e70a35efff6fa898500cfba19e73d00629702972f400a4f]: > {"message":"The security token included in the request is expired"}

tenor-216369781

Comment (regiskuckaertz) [reply data/database/b9573b35ed0df9269e70a35efff6fa898500cfba19e73d00629702972f400a4f]: haha 😅

**Comment by regiskuckaertz** [comment]
That failing preview test is surprising 🤔 How can I test it locally? I can't seem to find what the preview server is and how to debug it.

**Comment by regiskuckaertz** [comment]
@gtrufitt all clear now Sir

File: data/database/b9573b35ed0df9269e70a35efff6fa898500cfba19e73d00629702972f400a4f (hunk 1 above)
Comment (gtrufitt) [reply data/database/b9573b35ed0df9269e70a35efff6fa898500cfba19e73d00629702972f400a4f]: Did you still want this file (Does it matter?)

File: preview/test/PreviewServerTest.scala [hunk 2]
```diff
@@ -13,7 +13,7 @@ class PreviewTestSuite extends Suites (
   // These features are tested elsewhere, this is actually just here to ensure that the
   // preview server can start up and serve a page
 
-  "Preview Server" should "be able to serve an article" in goTo("/technology/2014/may/18/de-rosa-idol-bicycle-review-martin-love") { browser =>
-    browser.$("body").text should include ("Debating the pros and cons of each of these materials is the kind of thing that keeps passionate cyclists awake at night")
+  "Preview Server" should "be able to serve an article" in goTo("/music/2015/jan/01/music-awards-for-new-artists-blessing-or-curse") { browser =>
```
Comment (gtrufitt) [inline preview/test/PreviewServerTest.scala]: What happened here, for reference?

**Review by gtrufitt — APPROVED** [review]

File: preview/test/PreviewServerTest.scala (hunk 2 above)
Comment (regiskuckaertz) [reply preview/test/PreviewServerTest.scala]: I am not quite sure, testing locally in the browser worked well and I could preview that article. I tried replacing with several others and they all passed the test. I can dig further if needed, by printing out the hex dump of the response, but that will take some time.

**Comment by regiskuckaertz** [comment]
Ha, I didn't want to remove any file from the database in case it is needed. I'll drop that one 👍

File: preview/test/PreviewServerTest.scala (hunk 2 above)
Comment (gtrufitt) [reply preview/test/PreviewServerTest.scala]: I think it's fine, just odd though right!

**Pull request merged**